kafka-rest/src/main/java/io/confluent/kafkarest/entities/Broker.java

```diff
@@ -25,25 +25,36 @@
  */
 public final class Broker {
 
-  private int brokerId;
+  private final String clusterId;
```

Comment:
Would it make sense to hold a reference back to the cluster which owns this broker as opposed to the cluster id. It may(or may not) be nice (or one day be nice) for the broker to be able to introspect on its status within the cluster. i.e. isControler.

Reply:
I'm trying to keep the object-ref graph as acyclic as I can. We can add things like you said (e.g. isController()) later. I think it's possible to do it without adding a back-reference. But if one is needed I'm happy to add it. In the future. :)

Reply:
given the DI setup and how this is used, what @rigelbm has lgtm